### DISCUSSION TITLE
Actions added but not appearing in repo tab?

### DISCUSSION THREAD

**Discussion by barrymcgee** [community/community#26120] [Actions]
<p>I’m trying out Actions on my personal site before adding them to my company repos. </p>
<p>I’ve added two simple actions here;</p>
<p><a href="https://github.com/barrymcgee/barrymcgee.co.uk/tree/master/.github/workflows" rel="nofollow noopener">https://github.com/barrymcgee/barrymcgee.co.uk/tree/master/.github/workflows</a></p>
<ul>
<li>
<p>Greet everyone (example straight from official docs)</p>
</li>
<li>
<p>Assignee to reviewer (<a href="https://github.com/pullreminders/assignee-to-reviewer-action" rel="nofollow noopener">3rd party action</a>)</p>
</li>
</ul>
<p>From neither of these run nor do they appear in my repo “Actions” tab:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/2/28f64b2e47fb71cb700230ab123a900ce0516b68.png" data-download-href="/uploads/short-url/5QmMoyL2NT15mvFXL2YRqQiHyHK.png?dl=1" title="Screenshot 2019-08-23 at 11.24.47.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/505570/181095525-61192130-82e7-4b95-ab0c-f72276a3f1bb.png" alt="Screenshot 2019-08-23 at 11.24.47.png" data-base62-sha1="5QmMoyL2NT15mvFXL2YRqQiHyHK" width="690" height="371" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/2/28f64b2e47fb71cb700230ab123a900ce0516b68_2_690x371.png, https://aws1.discourse-cdn.com/github/optimized/2X/2/28f64b2e47fb71cb700230ab123a900ce0516b68_2_1035x556.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/2/28f64b2e47fb71cb700230ab123a900ce0516b68_2_1380x742.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/28f64b2e47fb71cb700230ab123a900ce0516b68_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2019-08-23 at 11.24.47.png</span><span class="informations">2384×1282 222 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>What gives? :face_with_monocle:</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/barrymcgee">@barrymcgee</a>,</p>
<p>Thanks for being here! Thanks for reaching out! Looks like you’ve not been enabled for the new version, therefore YAML wouldn’t run, for more info about migrating to YAML from HCL check out:  <a href="https://help.github.com/en/articles/about-github-actions#migrating-github-actions-from-hcl-to-yaml-syntax" rel="nofollow noopener">https://help.github.com/en/articles/about-github-actions#migrating-github-actions-from-hcl-to-yaml-syntax</a></p>
